kolibri/content/serializers.py

```diff
@@ -32,9 +34,22 @@ def to_representation(self, instance):
 
         return value
 
+    def get_lang_code(self, instance):
+        if instance.root.lang is None:
+            return None
```

Comment:
Question for @jonboiser is returning `None` here fine, and enforcing a default or other interpretation on the client side OK?

Reply:
If a channel has no language code, the only consequence is that it won't be filterable by language.